site/data/docs/custom-wallet-list.mdx

```diff
@@ -146,51 +151,51 @@ wallet.ledger(options: {
 #### MetaMask
 
 ```tsx
-import { wallet } from '@rainbow-me/rainbowkit';
+import { metaMaskWallet } from '@rainbow-me/rainbowkit/wallets';
 
-wallet.metaMask(options: {
+metaMaskWallet(options: {
   chains: Chain[];
   shimDisconnect?: boolean;
 });
 ```
 
-#### Rainbow
+#### Omni
```

Comment:
I noticed that the docs hadn't been updated with the Steakwallet to Omni rename, so I updated them as part of this.